### ISSUE TITLE
[IMPROVEMENT] Confused Upgradable badge

### ISSUE THREAD

**Issue by zgfg** [Androidacy/MagiskModuleManager#8]
When there is an Upgradable module, badge shows on the Online icon

But when you open Online tab, there is no list of Upgradable modules - confusing IMO

List of Upgradable modules is displayed in the Installed tab (I think that is correct)

But then please move the badge (number of Upgradable modules) from Online icon to Installed icon
![_Online](https://user-images.githubusercontent.com/51943147/236392619-96ff7732-9329-4251-a109-7b30908ac2b3.jpg)
![_Installed](https://user-images.githubusercontent.com/51943147/236392667-2153b45a-8f83-42b2-bb48-e179b4fe01c2.jpg)

**Comment by zgfg** [comment]
Yes - the module in my case was not available on the Repo but locally installed and upgradable by updateJson

I have just tested (latest Action build - I was testing the fix for crash on the module installation) the case when module is upgradable from Repo and in that case the Upgradable module is listed in the Online tab

**Comment by zgfg** [comment]
> I'm going to consider this issue closed even if by happy accident.

By which Action build it can be tested?

**Comment by Daviteusz** [comment]
> Upgradable modules are only supposed to be shown in online tab, but we have not experimented with that with modules only updatable via `updateJSON` so that probably needs tweaked

In my opinion, the module update badge should appear on the installed tab. The section of separated modules for updating is just in this tab. This is very confusing.

The badge on online should be when there is a new module in the repository (

Alternatively, the update section could be moved to the online tab.
